### PR TITLE
feat: refactor messages

### PR DESCRIPTION
**Pull request by foodaka**
Refactors warning message component to a dummy component where props are passed and live in parent

### CODE REVIEW

**Review by MareskoY — APPROVED** [review]

File: src/modules/markets/AssetsList.tsx [hunk 1]
```diff
@@ -105,12 +105,28 @@ export default function AssetsList() {
...
+        </Box>
+      )}
+
+      {currentNetworkConfig.name === 'Fantom' && (
+        <Box sx={{ mx: '24px' }}>
+          <MarketWarning
+            learnMore={true}
+            linkHref={`https://governance.aave.com/t/harmony-horizon-bridge-exploit-consequences-to-aave-v3-harmony/8614`}
+            warningMessage={
+              'Due to the Horizon bridge exploit, certain assets on the Harmony network are not at parity with Ethereum, which affects the Aave V3 Harmony market.'
```
Comment (defispartan) [inline src/modules/markets/AssetsList.tsx:127]: Wrong message

Comment (foodaka) [reply src/modules/markets/AssetsList.tsx]: thanks, nice catch `1979b5ea6214fadcea764c00a9740f8f5f7e8985`

File: src/components/transactions/Supply/SupplyModalContent.tsx [hunk 2]
```diff
@@ -220,15 +220,30 @@ export const SupplyModalContent = ({
         poolReserve.symbol === 'AAVE' &&
         isFeatureEnabled.staking(currentMarketData) && <AAVEWarning />}
       {poolReserve.symbol === 'SNX' && !maxAmountToSupply.eq('0') && <SNXWarning />}
-      {(currentNetworkConfig.name === 'Harmony' || currentNetworkConfig.name === 'Fantom') && (
+      {currentNetworkConfig.name === 'Harmony' ? (
```
Comment (defispartan) [inline src/components/transactions/Supply/SupplyModalContent.tsx:223]: All of the warnings in this file can be removed. This warning was in place when borrowing was disabled by supply was enabled in the Harmony market. If both markets are disabled, the supply modal is unreachable for all assets

File: src/modules/dashboard/lists/SupplyAssetsList/SupplyAssetsList.tsx [hunk 3]
```diff
@@ -166,13 +166,25 @@ export const SupplyAssetsList = () => {
         <>
           <Box sx={{ px: 6 }}>
             {currentNetworkConfig.name === 'Harmony' || currentNetworkConfig.name === 'Fantom' ? (
```
Comment (defispartan) [inline src/modules/dashboard/lists/SupplyAssetsList/SupplyAssetsList.tsx:168]: This can be cleaned up, no need to have two if statements on network name

**Comment by defispartan** [comment]
The warnings in AssetsList, SupplyAssetsList, and ReserveActions are not based on a condition check that the market is frozen. This shouldn't be merged until:

a.) Conditions are updated in these components
b.) The market is frozen

**Comment by bojanaave** [comment]
Looks fine!

**Comment by defispartan** [comment]
Superseded by #1113

**Pull request closed without merging**